src/components/layout/PanelChange/index.tsx

```diff
@@ -63,6 +65,14 @@ export const PanelChange: FC<IProps> = ({
     setIsLoading(mainLoading);
   }, [mainLoading]);
 
+  function getFormattedNumber(num: string) {
+    return parseFloat(num).toString().replace(/\B(?=(\d{3})+(?!\d))/g, ',');
+  }
+
+  function getFlowUSDValue(flow:string) {
+    return (parseFloat(flow as string) * coingeckoPrice).toFixed(2);
```

Comment:
if we arent going to do the K,M abbreviations, can we at least not show the decimal cents? Here, one has no cents after the decimal, another has just 1 number after the decimal, and another has 2 numbers after the decimal
<img width="172" alt="Screen Shot 2021-12-25 at 9 28 16 AM" src="https://user-images.githubusercontent.com/5077880/147388558-8653dd74-28c4-40ae-95e6-9cc100de80d5.png">